pan-domain-auth-verification/src/main/scala/com/gu/pandomainauth/service/CookieUtils.scala

```diff
@@ -45,7 +46,7 @@ object CookieUtils {
       user = User(firstName, lastName, email, data.get("avatarUrl")),
       authenticatingSystem = system,
       authenticatedIn = Set(authedIn.split(",").toSeq :_*),
-      expires = expires,
+      expires = Instant.ofEpochMilli(expires),
```

Comment:
Here we're reading _out_ of the cookie payload, and `expires` field in the Panda cookie payload is *milliseconds**.